sql/src/test/java/org/apache/druid/sql/calcite/NegativeTest.java

```diff
@@ -36,23 +37,52 @@
 import static org.junit.Assert.assertThrows;
 
 /**
- * Can be used to mark tests which are not-yet supported in decoupled mode.
+ * Can be used to mark tests which are not-yet supported for some reason.
+ *
+ * In case a testcase marked with this annotation fails - it means that the
+ * testcase no longer fails with the annotated expectation.
```

Comment:
Suggestion:

... no longer fails with the annotated expectation.  This means that a code change affected this test either 

1) it suddenly passes: yay, assuming it makes sense that it suddenly passes, remove the annotation and move on
2) it suddenly fails with a different error: validate that the new error is expected and either fix to continue failing with the old error or update the expected error.